src/ng/sanitizeUri.js

```diff
@@ -19,7 +20,8 @@ function $$SanitizeUriProvider() {
    * Any url about to be assigned to a[href] via data-binding is first normalized and turned into
    * an absolute url. Afterwards, the url is matched against the `aHrefSanitizationWhitelist`
    * regular expression. If a match is found, the original url is written into the dom. Otherwise,
-   * the absolute url is prefixed with `'unsafe:'` string and only then is it written into the DOM.
+   * the absolute url is prefixed with `'unsafe:'` string and only then is it written into the DOM,
+   * making it inactive.
```

Comment:
Could you maybe document that those are used in the $sce.URL sanitization now, so it's reflected in the API docs ? (and same below)